rain_api_core/urs_util.py

```diff
@@ -294,8 +294,19 @@ def do_login(args, context, cookie_domain=''):
         else:
             redirect_to = get_base_url(context)
 
+        if 'user_groups' not in user_profile:
+            user_profile['user_groups'] = []
+
+        jwt_cookie_payload = {
+            'urs-user-id': user_id,
+            'urs-access-token': auth['access_token'],
+            'urs-groups': user_profile['user_groups'],
+            'created': time(),
```

Comment:
If we're doing `created` instead of `expires`, then its implicitly up to the decoding APP to decide how long to trust a cookie for?

Reply:
I can think of reasons why we may want to know how old a cookie is, an expired field doesn't tell us that. There's already an 'expires' property inherent to cookies.

Reply:
Turns out there's a registered claim for jwt `exp`, that will fail the validation of the cookie when the current date passes it.